### ISSUE TITLE
Intermittent UI test failure - < SettingsSearchTest.verifySearchEnginesWithRTLLocale>

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#28257]
### Stacktrace:
java.lang.AssertionError: Failed to perform isDisplayed check.
Reason: Expected exactly '1' node but could not find any node that satisfies: (Text + EditableText contains 'Google' (ignoreCase: false))

	at androidx.compose.ui.test.SemanticsNodeInteraction.fetchOneOrDie(SemanticsNodeInteraction.kt:145)
	at androidx.compose.ui.test.SemanticsNodeInteraction.fetchSemanticsNode(SemanticsNodeInteraction.kt:79)
	at androidx.compose.ui.test.AndroidAssertions_androidKt.checkIsDisplayed(AndroidAssertions.android.kt:29)
	at androidx.compose.ui.test.AssertionsKt.assertIsDisplayed(Assertions.kt:33)
	at org.mozilla.fenix.ui.robots.SearchRobot.verifySearchEngineShortcuts(SearchRobot.kt:291)
	at org.mozilla.fenix.ui.SettingsSearchTest$verifySearchEnginesWithRTLLocale$8.invoke(SettingsSearchTest.kt:417)
	at org.mozilla.fenix.ui.SettingsSearchTest$verifySearchEnginesWithRTLLocale$8.invoke(SettingsSearchTest.kt:415)
	at org.mozilla.fenix.ui.robots.HomeScreenRobot$Transition.openSearch(HomeScreenRobot.kt:520)
	at org.mozilla.fenix.ui.SettingsSearchTest.verifySearchEnginesWithRTLLocale(SettingsSearchTest.kt:415)

    
### Build: 12/19

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-22166)

**Comment by Mugurell** [comment]
Seen it failing two times in https://github.com/mozilla-mobile/fenix/pull/28224.
Also fails locally.

**Comment by Mugurell** [comment]
Will disable the test to unblock landing.

**Comment by AndiAJ** [comment]
Closing in favour of #28182
